### PR TITLE
Add API wrapper for sphere generator

### PR DESCRIPTION
**Pull request by eitanlees**
I noticed the sphere generator was missing from the data gen wrappers.

### CODE REVIEW

File: altair/vegalite/v3/api.py [hunk 1]
```diff
@@ -1984,4 +1985,12 @@ def graticule(**kwds):
...
\ No newline at end of file
+  return core.GraticuleGenerator(graticule=graticule)
+
+
+def sphere(**kwds):
+  """Sphere generator."""
+  if not kwds:
+      # sphere: True indicates default parameters
+      sphere = True
+  return core.SphereGenerator(sphere=sphere, **kwds)
```
Comment (jakevdp) [inline altair/vegalite/v3/api.py]: Possible for ``sphere`` to be undefined here.

File: altair/vegalite/v3/tests/test_api.py [hunk 2]
```diff
@@ -858,4 +858,9 @@ def test_graticule():
...
 
     data = alt.graticule(step=[15, 15])
-    assert data.to_dict() == {'graticule': {'step': [15, 15]}}
\ No newline at end of file
+    assert data.to_dict() == {'graticule': {'step': [15, 15]}}
+
+
+def test_sphere():
+    data = alt.sphere()
+    assert data.to_dict() == {'sphere': True}
```
Comment (jakevdp) [inline altair/vegalite/v3/tests/test_api.py]: Test passing of kwargs as well - anything is allowed according to the schema, and ``alt.sphere(key='val').to_dict()`` should result in ``{'sphere': {'key': 'val'}}``

**Comment by eitanlees** [comment]
Thanks for the review! I have a question. After looking a little closer at the documentation for the [Sphere Generator](https://vega.github.io/vega-lite/docs/data.html#sphere) it seems that the generator doesn't take any additional parameters.

Should `alt.sphere` take no parameters? Should it take in `**kwds` and then raise an error if any are passed? 

Thanks again for any advice.

**Comment by jakevdp** [comment]
Ah, you're right: the ``sphere`` parameter of a ``SphereGenerator`` definition can be either the boolean True or an empty dict – I'd missed the ``additionalProperties: false`` when I looked before.

https://github.com/altair-viz/altair/blob/2d74e4a189b61e051da2d453357d38b5e8016106/altair/vegalite/v3/schema/vega-lite-schema.json#L10781-L10795

So yeah, the ``sphere()`` function should probably just take no keywords.

**Comment by eitanlees** [comment]
Alright, done and done

**Comment by jakevdp** [comment]
Looks good, thanks!

**Pull request merged**